module/Application/src/Application/View/Helper/OrganizationNameHelper.php

```diff
@@ -0,0 +1,45 @@
+<?php 
+
+namespace Application\View\Helper;
+
+use Zend\View\Helper\AbstractHelper;
+use Zend\ServiceManager\ServiceLocatorAwareInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use People\Entity\Organization;
+
+class OrganizationNameHelper extends AbstractHelper implements ServiceLocatorAwareInterface{
```

Comment:
Perché in questo Helper viene recuperata ancora l'organization quando è già stata recuperata dal controller?